3-terrarium/2-intro-to-css/translations/README.pt-br.md

```diff
@@ -0,0 +1,268 @@
+# Projeto Terrário Parte 2: Introdução a CSS
+
+![Introdução a CSS](../../../sketchnotes/webdev101-css.png)
```

Comment:
@skyeenicole

Reply:
rytola536@gmail.com
![79C5B8D6-A04D-44F4-8F0D-B3C1BF11B3B8](https://github.com/microsoft/Web-Dev-For-Beginners/assets/143288017/f99463fc-9aa9-43cf-a1eb-489d214b8858)

Reply:
??? You can help yes?